spec/models/program_spec.rb

```diff
@@ -58,4 +58,31 @@
       expect(conference.program.difficulty_levels.count).to eq 3
     end
   end
+
+  describe 'validation' do
+    it '#check_languages_format' do
```

Comment:
@Ana06 please write for each test case a separate test because otherwise it's hard to debug if one test fails!